docs/content/en/docs/topics/repair/_index.md

```diff
@@ -22,21 +22,30 @@ The requirement for your environment may vary considerably, however the general
 1. Kubernetes cluster with the following elements deployed:
    * [K8ssandra]({{< ref "getting-started#install-k8ssandra" >}}) Helm chart
    * [Ingress Controller]({{< ref "ingress" >}})
-2. DNS name configured for the Reaper interface, referred to as _REPAIR DOMAIN_ below.
+2. DNS name configured for the Reaper web interface
+
+{{% alert title="Tip" color="success" %}}
+As an alternative to configuring an Ingress, consider port forwarding. It's another way to provide external access to resources that have been deployed by K8ssandra in your Kubernetes environment. Those resources could include Prometheus metrics, pre-configured Grafana dashboards, and the Reaper web interface for repairs of Cassandra&reg; data. The `kubectl port-forward` command does not require an Ingress/Traefik to work. 
+
+* Developers, see [Set up port forwarding]({{< ref "/docs/getting-started/developer/#set-up-port-forwarding" >}}).  
+* Site reliability engineers, see [Configure port forwarding]({{< ref "/docs/getting-started/developer/#set-up-port-forwarding" >}}).
+{{% /alert %}}
 
 ## Access the Reaper web interface
 
 ![Reaper UI](reaper-main-ui.png)
 
 With the prerequisites satisfied the Reaper web interface should be available at the following address:
 
-http://REPAIR_DOMAIN/webui
+http://REAPER_DOMAIN/webui
 
-For example, to upgrade a previously installed `k8ssandra` that's running locally, where the `releaseName` used on the prior `helm install` command was `k8ssandra`:
+For example, to upgrade a previously installed `k8ssandra` that's running locally, where the `cluster-name` used on the prior `helm install` command was `k8ssandra`:
```

Comment:
The value you specify to `helm install` is releaseName, not clusterName. I'd avoid representing either with hyphenated values; using them in your release or cluster name leads to pod/service/etc names that are somewhat more difficult for humans to parse.

(clusterName defaults to be the same as releaseName if you don't specify something else)

Reply:
Incorporated, thanks Jake.